source/MaterialXRender/SampleObjLoader.h

```diff
@@ -0,0 +1,62 @@
+//
+// TM & (c) 2017 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#ifndef MATERIALX_SAMPLEOBJLOADER_H
+#define MATERIALX_SAMPLEOBJLOADER_H
+
+/// @file
+/// Sample OBJ geometry loader
+
+#include <MaterialXRender/GeometryHandler.h>
+
+namespace MaterialX
+{
+
+/// Shared pointer to an SampleObjLoader
```

Comment:
Is there still a need for the SampleObjLoader, now that TinyObjLoader is available by default in MaterialXRender and MaterialXView?  If there's still unique functionality in SampleObjLoader, would it be possible to port it over to TinyObjLoader as an option?

Reply:
It's used for all the additional "fake" streams of data to test multiple sets of geom streams and geomattrs for the test suite. Maybe it can be added as a base class utility but I haven't looked at it how much work it would be.